jira-stats-serverless/analyzer/domain/dev_time_calculator.go

```diff
@@ -7,6 +7,12 @@ import (
 
 const StateDev = "In Development"
 
+type Now func() time.Time
+
+type DaysCalculator struct {
+	ClockNow Now
```

Comment:
Is it a kind of idiomatic way of defining interfaces in Go? :D

Reply:
Can you bind here `Time.Now` as a default implementation? In that way we could get rid of [those](https://github.com/VirtusLab/jira-stats/pull/4/files#diff-1c8c9bc5953f332b161f877d5e02bebbR145) `if`'s

Reply:
There is no such thing a default implementation in Golang ( https://stackoverflow.com/questions/37135193/how-to-set-default-values-in-go-structs ) but it might be possible to hack something